velox/dwio/parquet/writer/Writer.h

```diff
@@ -109,6 +109,13 @@ struct WriterOptions : public dwio::common::WriterOptions {
   std::optional<std::string> parquetWriteTimestampTimeZone;
   bool writeInt96AsTimestamp = false;
 
+  enum class ParquetDataPageVersion {
+    V1,
+    V2,
+  };
+
+  std::optional<ParquetDataPageVersion> parquetDataPageVersion;
```

Comment:
I just realized there are only 2 values. Let's keep this a `bool useParquetDataPageV2`. The default will be `false`.

Reply:
Refactored.

Reply:
Ok, refactored to accommodate boolean instead.